webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -55,13 +55,14 @@ function TeamController(props: Props) {
 
     useEffect(() => {
         InitialLoadingScreen.stop();
-        async function fetchInitialChannels() {
-            await props.fetchAllMyTeamsChannelsAndChannelMembersREST();
+        async function fetchAllChannels() {
+            await props.fetchAllMyTeamsChannels();
 
             setInitialChannelsLoaded(true);
         }
 
-        fetchInitialChannels();
+        props.fetchAllMyChannelMembers();
```

Comment:
I'm a bit leery of this change to not wait for channel members to load before settings `initialChannelsLoaded` to true. I get that you want to not have the initial load be worse for users that already have a lot of channel members, but this change may affect people who didn't run into this issue before, so I think we might want to hold off on it until we can verify that this fixes the linked issues. I'm also trying to avoid any chance that the rest of the PR has to be reverted if we find that this causes a bug. 

I'm maybe 2/5 on that at most though. Thoughts?

Reply:
The impact on initial load times if we wait for channels members to load is massive, especially for those that this bug affects. I don't have exact measurements but I did a rough count and the time was around doubled, which I don't think is acceptable.

I'd be willing to have to revert this PR if we do find that there's a bug (or just change the behaviour to wait for it) because I think the bug fix here is important, unless we know of a concrete scenario where this will fail. In testing, this seems to work fine.

Reply:
Okay, I think that should be fine. I am still a bit leery of this when I'd potentially blame the bug that's being addressed by your changes on incorrect optimization, but I can definitely see your point here because this sort of loop to load channel members was what caused Jason massively inflated load times with GraphQL enabled